ios/Classes/HyperWebviewFlutterPlugin.m

```diff
@@ -0,0 +1,106 @@
+#import "HyperWebviewFlutterPlugin.h"
+
+@interface HyperWebviewFlutterPlugin()
+
+@property (nonatomic, strong) FlutterMethodChannel* dartChannel;
+@property (nonatomic, strong) FlutterMethodChannel* nativeChannel;
+
+@end
+
+const int OPENAPPS_REQUEST_CODE = 19;
+const int GET_RESOURCE_NAME = 453;
+const int CAN_OPEN_APP_CODE = 789;
+@implementation HyperWebviewFlutterPlugin
+
+
++ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
```

Comment:
@sandeepsj  Why did we decide to use ObjectiveC here? Why not swift?
We are thinking of moving to Swift everywhere. Let's use swift in this new repo

Reply:
@Subhash703 - to take care in next PR if still open